src/binfhe/include/lwe-cryptoparameters.h

```diff
@@ -68,6 +68,8 @@ class LWECryptoParams : public Serializable {
                              const NativeInteger& q_KS, double std, uint32_t baseKS,
                              SecretKeyDist keyDist = UNIFORM_TERNARY)
         : m_q(q), m_Q(Q), m_qKS(q_KS), m_n(n), m_N(N), m_baseKS(baseKS), m_keyDist(keyDist) {
+        if(m_q == NativeInteger(0))
```

Comment:
Technically, none of the parameters n, N, q, Q, q_KS, baseKS should be zero. Can you please throw exceptions for them too?

Reply:
The changes have been made and all CI jobs have succeeded